app/src/main/java/com/android/unio/ui/association/AssociationProfile.kt

```diff
@@ -393,6 +396,20 @@ private fun AssociationMembers(
                   user.value?.lastName?.let {
                     val lastName = it
                     Text("${firstName} ${lastName}")
```

Comment:
A small detail, but the curly braces are not needed here

Reply:
true ! I think it was because before it was a user.firstName ; )